### ISSUE TITLE
I downloaded simple CSV file of data and the columns don't line up and so don't make any sense.  Am I supposed to post-process it some how after download?

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#4177]
**I downloaded simple CSV file of data and the columns don&#39;t line up and so don&#39;t make any sense. Am I supposed to post-process it some how after download?**

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/6bcadcf8e5446aa0b3d951cdf0ec1993:2ba092ec28983a17fc0fb5e22da4caf17808661783e8cc551b9dbaf4956b132c6859fc741bd3f6ef0f0090f7706d6a913efcf07bcd216dd7932f05efb981caf8e9b840566a523bfbd8055d65db6a4a4ad353251ebf876812f3fab9dc1b4ca527) - [Send email](https://www.gbif.org/api/feedback/user/mailto/6bcadcf8e5446aa0b3d951cdf0ec1993:2ba092ec28983a17fc0fb5e22da4caf17808661783e8cc551b9dbaf4956b132c6859fc741bd3f6ef0f0090f7706d6a913efcf07bcd216dd7932f05efb981caf8e9b840566a523bfbd8055d65db6a4a4ad353251ebf876812f3fab9dc1b4ca527)
System: Chrome 92.0.4515 / Windows 10.0.0
Referer: https://www.gbif.org/user/profile
Window size: width 1280 - height 824
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2022-07-20T06:51:56.611Z',to:'2022-07-20T06:57:56.611Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2022-07-20T06:51:56.611Z',to:'2022-07-20T06:57:56.611Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by MortenHofft** [comment]
Thanks for getting in touch

Assuming you generated your download yourself you should have received an email explaining this.

Please let me know if the reason for your question is because you missed the information in the email or because the explanation is hard to follow. That will help us understand what to adjust. Thanks

The email instructs you to read this: https://www.gbif-uat.org/faq?question=opening-gbif-csv-in-excel

> user informed per email
